enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterRule.java

```diff
@@ -153,4 +242,71 @@ protected void after()
             throwable.printStackTrace();
         }
     }
+
+    public File directory( String name )
+    {
+        return testDirectory.directory( name );
+    }
+
+    public File cleanDirectory( String name ) throws IOException
+    {
+        return testDirectory.cleanDirectory( name );
+    }
+
+    /**
+     * Adapter for providing a static config value into a setting where per-instances dynamic config values
+     * are supplied.
+     *
+     * @param value static config value.
+     * @return this {@link ClusterRule} instance, for builder convenience.
+     */
+    public static IntFunction<String> constant( String value )
+    {
+        return ClusterManager.constant( value );
+    }
+
+    /**
+     * Dynamic configuration value, of sorts. Can be used as input to {@link #config(Setting, IntFunction)}.
+     * Some configuration values are a function of server id of the cluster member and this is a utility
+     * for creating such dynamic configuration values.
+     *
+     * @param valueBeforeAddingOneBasedServerId value onto which one-based server id is added. So for example
+     * a value of 10 would have cluster member with server id 2 that config value set to 12.
+     */
+    public static IntFunction<String> intBase( final int valueBeforeAddingOneBasedServerId )
```

Comment:
imho if function name will mention that it's server id based it would be more clear.
Like `serverIdBased` for example.